docs/azurefunction_maven.md

```diff
@@ -0,0 +1,25 @@
+# Config Maven for VS Code
```

Comment:
Nice document. I think we can also reference it in the README: https://github.com/Microsoft/vscode-azurefunctions#java